colortv/docs/androidTVSDK.md

```diff
@@ -591,83 +591,214 @@ public class MainActivity extends Activity {
-| ctv_tvDescription          | TextView     |                                     | Display description                                       | ALL    |
-| ctv_tvDuration             | TextView     |                                     | Display duration                                          | ALL    |
-| ctv_tvGenre                | TextView     |                                     | Define how each genre should look like. Visibility must be GONE                   | ALL    |
-| ctv_llGenres               | LinearLayout |                                     | Each ctv_tvGenre will be added here                       | ALL    |
+This method is used to set a custom item layout for specified device type. You can add additional images, texts etc. All views are animated if they contain selectors with the default state and state_selected (TV only).
+
+<details><summary>Available IDs</summary>
+
+
+#### **ID:** ctv_hide
+* **View type:** Any view extending the View class (eg. `LinearLayout`, `ImageView`)
+* **Description:** Used to hide views on focus and show them on unfocus. Use any layout if you would like to hide more than one view.
+* **Animation:** Hides (`Visibility.GONE`) the view and its child views on focus and shows them on unfocus.
+* **Device:** TV
+<br /><br />
+
+
+#### **ID:** ctv_show
+* **View type:** Any view extending the View class (eg. `LinearLayout`, `ImageView`)
+* **Description:** Used to show views on focus and hide them on unfocus. Use any Layout if you would like to show more than one view.
+* **Animation:** Shows (`Visibility.VISIBLE`) the view and its child views on focus and hides them on unfocus.
+* **Device:** TV
+<br /><br />
+
+
+#### **ID:** ctv_autoPlayTimerContainer
+* **View type:** `FrameLayout`
+* **Description:** Used to inject countdown timer to automatically start playing the first recommended video
+* **Animation:** Hides on any interaction on items list or when timer reaches 0.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_videoPreviewContainer
+* **View type:** Any view extending the ViewGroup class (eg. `LinearLayout`, `RelativeLayout`)
+* **Description:** Used to inject our `VideoPlayer` for previews playback. Check our [custom layout](https://github.com/color-tv/android-SampleApp/blob/master/SampleApp/app/src/main/res/layout/custom_item_layout_2.xml) if you would like to oversize the preview video.
+* **Animation:** Plays the preview and hides the thumbnail on focus (only if the preview is available) and stops playing the preview and shows the thumbnail on unfocus.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_ivVideoThumbnail
+* **View type:** `ImageView`
+* **Description:** Used to display the video thumbnail.
+* **Animation:** Hides on focus if there is preview available and shows on unfocus.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_ivAppLogo
+* **View type:** `ImageView`
+* **Description:** Used to display logo.
```

Comment:
I'd write: Used to display your app's logo.
To point out what kind of logo it is.